docs/api/memory/MemTrace.md

```diff
@@ -24,7 +24,7 @@ For a step-by-step guide about how to use optimize memory using runtime memory t
 
 ## Memory tracing example
 
-[![View code](https://www.mbed.com/embed/?url=https://os.mbed.com/teams/mbed_example/code/memory_tracing_example/)](https://os.mbed.com/teams/mbed_example/code/memory_tracing_example/file/168ab14e6694/main.cpp)
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_Platform/Memory_tracing_ex_1/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/APIs_Platform/Memory_tracing_ex_1/main.cpp)
```

Comment:
I don't see this change in the mbed-os-examples-docs_only PR.

Reply:
example is here https://github.com/ARMmbed/mbed-os-examples-docs_only/pull/88